front/src/routes/scene/edit-scene/actions/SendSms.jsx

```diff
@@ -0,0 +1,40 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import { Text } from 'preact-i18n';
+
+import TextWithVariablesInjected from '../../../../components/scene/TextWithVariablesInjected';
+
+class SendSms extends Component {
+  updateText = text => {
+    this.props.updateActionProperty(this.props.columnIndex, this.props.index, 'text', text);
+  };
+
+  render(props, {}) {
+    return (
+      <div>
+        <div class="form-group">
+          <label class="form-label">
+            <Text id="editScene.actionsCard.smsSend.textLabel" />{' '}
+            <span class="form-required">
+              <Text id="global.requiredField" />
+            </span>
+          </label>
+          <div class="mb-1 small">
+            <Text id="editScene.actionsCard.smsSend.explanationText" />
+          </div>
+          <div className="tags-input">
+            <TextWithVariablesInjected
```

Comment:
Missing a placeholder, it looks a bit empty: 

![Screenshot 2024-11-22 at 09 59 00](https://github.com/user-attachments/assets/6677c72b-b060-4d1c-9ad8-fece95fefa28)

Reply:
It will be less empty now.
![image](https://github.com/user-attachments/assets/3dee0851-6786-48f9-8f74-62fa95b9f008)